tabular/tests/unittests/test_tabular.py

```diff
@@ -447,6 +447,49 @@ def test_tabularHPO():
                            seed_val=seed_val, fit_args=fit_args)
 
 
+@pytest.mark.slow
+def test_tabular_feature_prune():
+    ############ Benchmark options you can set: ########################
+    perf_threshold = 1.1  # How much worse can performance on each dataset be vs previous performance without warning
+    seed_val = 99  # random seed
+    subsample_size = None
+    ag_args = {
+        'feature_prune_kwargs': {
+            'stop_threshold': 3,
+            'prune_ratio': 0.05,
+            'prune_threshold': None,
+            'n_train_subsample': 10000,
+            'n_fi_subsample': 5000,
+            'min_fi_samples': 10000,
```

Comment:
smaller values might make this test run quicker? is there any advantage to using bigger values here?

Reply:
Good point - no particular reason, I just ensured that pruning logic does trigger and removes some feature in one of the test dataset and the time limit is not too long (~15s). I will see what happens when I lower this number